### PR TITLE
DEVDOCS-2932 Update documentation for DELETE /settings/email-statuses

### PR DESCRIPTION
**Pull request by markcmurphy**
[DEVDOCS-2932](https://jira.bigcommerce.com/browse/DEVDOCS-2932)

Add specs from the updated `settings.v3.yml` file.

### CODE REVIEW

**Review by funivan — APPROVED** [review]

File: reference/settings.v3.yml [hunk 1]
```diff
@@ -769,16 +769,58 @@ paths:
...
+                product_review_email:
+                  nullable: true
+            examples:
+              Reset gift certificate template:
+                value:
+                  gift_certificate_email: null
+              Reset product review and invoice email templates:
+                value:
+                  invoice_email: null
+                  product_review_email: null
```
Comment (traci707) [inline reference/settings.v3.yml:823]: description: Provide email types that should be deleted

**Review by traci707 — COMMENTED** [review]
Looks good. If you like you can add the following line.

description: Provide email types that should be deleted

File: reference/settings.v3.yml (hunk 1 above)
Comment (markcmurphy) [reply reference/settings.v3.yml:823]: @traci707 That is still listed in the description on line 773. Are you referring to another field?

Comment (markcmurphy) [reply reference/settings.v3.yml:823]: @traci707 the description for the `DELETE` method is on 773, is that what you're referring to?

Comment (traci707) [reply reference/settings.v3.yml:823]: There were two descriptions in the swagger file supplied by the engineer. It is optional.

**Review by traci707 — APPROVED** [review]

**Pull request merged**